pkg/sql/catalog/systemschema/system.go

```diff
@@ -1338,10 +1338,11 @@ CREATE TABLE public.inspect_errors (
     id OID NOT NULL,
     primary_key STRING NULL,
     details STRING NOT NULL,
+    crdb_internal_expiration TIMESTAMPTZ NOT NULL DEFAULT current_timestamp():::TIMESTAMPTZ + '90 days':::INTERVAL ON UPDATE current_timestamp():::TIMESTAMPTZ + '90 days':::INTERVAL,
```

Comment:
this column should be marked as `NOT VISIBLE` also, since that's the expected behavior when using `ttl_expire_after`.